Formula/c/clickhouse-sql-parser.rb

```diff
@@ -0,0 +1,19 @@
+class ClickhouseSqlParser < Formula
+  desc "Writing clickhouse sql parser in pure Go"
+  homepage "https://github.com/AfterShip/clickhouse-sql-parser"
+  url "https://github.com/AfterShip/clickhouse-sql-parser/archive/refs/tags/v0.3.8.tar.gz"
+  sha256 "dda8b9503bf0e71d78fce260dcf3c72b2929580d554a7e4d03bbb4177c63868b"
+  license "MIT"
+  head "https://github.com/AfterShip/clickhouse-sql-parser.git", branch: "master"
+
+  depends_on "go" => [:build, :test]
```

Comment:
Is `go` needed at runtime? Seems odd to need it for `:test`.

Reply:
Yes, we don't need it for `:test`. Will remove it.